### DISCUSSION TITLE
Telegram bot to remove exif data of photos and videos

### DISCUSSION THREAD

**Discussion by theStratus** [WeAreMahsaAmini/Core#422] [Ideas]
Dear all

People share the photos and videos of protests as files so that quality is preserved on social media (telegram). However, these files have exif data that may identify the owner. A telegram bot that receives these files, removes the exif data and returns them anonymous is beneficial. 
Is there anyone who can do that or introduce such a bot?

**Comment by 0ne-zero** [comment]
I agree with you, I've created this [repo](https://github.com/0ne-zero/ExifRemoverBot) for it.
But for now, it just can remove EXIF metadata from JPEG and PNG.
I Used Go for that, but we can use any tools for removing EXIF data, If somebody knows that there is something stronger than this program, we can use it in the program, and use it to remove EXIF.
BTW, we can just make this program a proxy and send the sent file to the online remover website, BUT first of all, we should make sure the website is safe. I said that because I tested and realized that some online remover EXIF website removes more data than our telegram bot

**Reply by theStratus** [reply]
Thanks for the contribution. Let's see how it goes on. 
I agree with you on safety. I trust known websites more than personal hosting.
